packages/lit-element/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "lit-element",
+  "version": "3.0.0-pre.0",
+  "description": "A simple base class for creating fast, lightweight web components",
+  "license": "BSD-3-Clause",
+  "homepage": "https://lit-element.polymer-project.org/",
+  "repository": "Polymer/lit-element",
+  "type": "module",
+  "main": "lit-element.js",
+  "module": "lit-element.js",
+  "directories": {
+    "test": "test"
+  },
+  "files": [
+    "/lib/",
+    "/src/",
+    "!/src/demo/",
+    "!/src/test/",
+    "/lit-element.d.ts",
+    "/lit-element.d.ts.map",
+    "/lit-element.js",
+    "/lit-element.js.map",
+    "/ts3.4/"
+  ],
+  "scripts": {
+    "build": "tsc && rm -rf ./ts3.4 && downlevel-dts . ts3.4 && cp tsconfig.json ./ts3.4/",
+    "build:watch": "tsc --watch",
+    "build:babel-test": "babel src/test/lib/decorators_test.ts --out-file test/lib/decorators-babel_test.js",
+    "gen-docs": "typedoc src/lit-element.ts src/lib/updating-element.ts",
+    "test": "npm run build && npm run build:babel-test && wct",
```

Comment:
If you want to get some of the tests running on initial checkin, like just the updating-element and lit-element core tests, the upgrade to web-test-runner was pretty easy, and it's very, very fast.

Reply:
Done.